app/PrintAccount.php

```diff
@@ -36,17 +33,18 @@ public function hasEnoughMoney($balance) {
         return $this->balance >= abs($balance);
     }
 
-    public static function getCost($pages, $twosided, $number_of_copies = 1) {
-        if ($twosided) {
-            $cost;
-            if ($pages % 2 == 0) {
-                $cost = $pages / 2 * PrintAccount::COST['twosided'];
-            } else {
-                $cost = ($pages  - 1) / 2 * PrintAccount::COST['twosided'] + $pages * PrintAccount::COST['onesided'];
-            }
-            return $cost * $number_of_copies;
-        } else {
-            return $pages * PrintAccount::COST['onesided'] * $number_of_copies;
-        }
+    public static function getCost($pages, $is_two_sided, $number_of_copies = 1) {
+        if (!$is_two_sided)
+            return $pages * self::$COST['one_sided'] * $number_of_copies;
+    
+        $orphan_ending = $pages % 2;
+        $one_copy_cost = floor($pages / 2) * self::$COST['two_sided']
+            + $orphan_ending * self::$COST['one_sided'];
+        return $one_copy_cost * $number_of_copies;
     }
 }
+
+PrintAccount::$COST = [
+    'one_sided' => env('PRINT_COST_ONESIDED'),
+    'two_sided' => env('PRINT_COST_TWOSIDED'),
+];
```

Comment:
Shouldn't we have newline ending?

Reply:
Could you elaborate?

Reply:
Now I can not check it but if you use the PhpStorm auto formatter, that would insert an ending newline character at the very end of the file. Since it is missing now, I assumed you haven't used the auto formatting. Its nothing just a comment, that is why approved this change already.

Reply:
I do not set up the autoformatter until we agree in a universal convention. However, I understand your problem, added a new line.